website/docs/r/lb_listener.html.markdown

```diff
@@ -85,6 +85,67 @@ resource "aws_lb_listener" "front_end" {
 }
 ```
 
+### Authenticate-cognito Action
+
+```hcl
+resource "aws_lb" "front_end" {
+  # ...
+}
+
+resource "aws_cognito_user_pool" "pool" {
+  # ...
+}
+
+resource "aws_cognito_user_pool_client" "client" {
+  # ...
+}
+
+resource "aws_cognito_user_pool_domain" "domain" {
+  # ...
+}
+
+resource "aws_lb_listener" "front_end" {
+  load_balancer_arn = "${aws_lb.front_end.arn}"
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
+    type = "authenticate-cognito"
+    authenticate_cognito {
+      "user_pool_arn"       = "${aws_cognito_user_pool.pool.arn}"
+      "user_pool_client_id" = "${aws_cognito_user_pool_client.client.id}"
+      "user_pool_domain"    = "${aws_cognito_user_pool_domain.domain.domain}"
+    }
+  }
+}
+```
+
+### Authenticate-oidc Action
+
+```hcl
+resource "aws_lb" "front_end" {
+  # ...
+}
+
+resource "aws_lb_listener" "front_end" {
+  load_balancer_arn = "${aws_lb.front_end.arn}"
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
```

Comment:
Same here, I believe this requires two actions.

Reply:
2 (default_)actions have been added. Should be 🙆‍♂️ now.